profile/deploy.yml

```diff
@@ -289,9 +288,9 @@ services:
       OR_IDENTITY_SESSION_OFFLINE_TIMEOUT_MINUTES: ${OR_IDENTITY_SESSION_OFFLINE_TIMEOUT_MINUTES:-2628000}
 
       # Enable/disable Prometheus metrics collection
-      OR_METRICS_ENABLED: ${OR_METRICS_ENABLED:-true}
+      #OR_METRICS_ENABLED: ${OR_METRICS_ENABLED:-true}
       # Web server port used to host metrics endpoint
-      OR_METRICS_PORT: ${OR_METRICS_PORT:-8404}
+      #OR_METRICS_PORT: ${OR_METRICS_PORT:-8405}
```

Comment:
(same comment regards to the `8404` to `8405` change)

Reply:
Again same as above